### DISCUSSION TITLE
Couldn't find "openssl/core_names.h" after "make release"

### DISCUSSION THREAD

**Discussion by kolik29** [OvenMediaLabs/OvenMediaEngine#1111] [Help]
I start command make release and get this error:
`####################################################
#                                                  #
#           AirenSoft Make System v0.63            #
#                                                  #
####################################################
   - Operating system        : linux
   - Core count              : 4
   - GCC version             : 8.3.0
   - Build option            : RELEASE
   - Paths for linker        : /opt/ovenmediaengine/lib:/opt/ovenmediaengine/lib64
   - Paths for pkg-config    : /opt/ovenmediaengine/lib/pkgconfig:/opt/ovenmediaengine/lib64/pkgconfig
   - Total Projects count    : 54

[  1/422|  0%] Preparing directories...
[  2/422|  0%] [static] libovcrypto.a: C++ projects/base/ovcrypto/certificate.cpp => intermediates/RELEASE/objs/base/ovcrypto/certificate.o
projects/base/ovcrypto/certificate.cpp:3:10: fatal error: openssl/core_names.h: No such file or directory
 #include <openssl/core_names.h>
          ^~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make: *** [core/build_objects.mk:21: intermediates/RELEASE/objs/base/ovcrypto/certificate.o] Error 1`

OS Debian
OME 0.14.18

I use this instruction https://airensoft.gitbook.io/ovenmediaengine/getting-started and get error after use command  "make release". I try this command `apt install openssl libssl-dev`, but it doesn't work. What do i need to install?

**Comment by bchah** [comment]
Did you run '/misc/prerequisites.sh' first? It seems like the library for openssl is missing which implies the prerequisites script has not been run.

**Comment by kolik29** [comment]
> Did you run '/misc/prerequisites.sh' first? It seems like the library for openssl is missing which implies the prerequisites script has not been run.

```
This program [./prerequisites.sh] does not support your operating system [Debian GNU/Linux]
Please refer to manual installation page
+ recheck=false
+ [ x = x--recheck ]
+ AUTOCONF=autoconf
+ AUTOMAKE=automake
+ ACLOCAL=aclocal
+ AUTOHEADER=autoheader
+ mkdir -p autoconf autoconf/helpers config
+ automake --print-libdir
./autogen.sh: 1: ./autogen.sh: automake: not found
+ autolib=
+ test ! x = x
+ mv -f autoconf/aclocal.m4 autoconf/aclocal.m4.old
mv: cannot stat 'autoconf/aclocal.m4': No such file or directory
+ mkdir -p autoconf/m4.old autoconf/m4
+ mv -f autoconf/m4/pa_add_cflags.m4 autoconf/m4/pa_add_flags.m4 autoconf/m4/pa_add_headers.m4 autoconf/m4/pa_add_ldflags.m4 autoconf/m4/pa_arg_bool.m4 autoconf/m4/pa_arg_disabled.m4 autoconf/m4/pa_arg_enabled.m4 autoconf/m4/pa_c_typeof.m4 autoconf/m4/pa_check_bad_stdc_inline.m4 autoconf/m4/pa_func_attribute.m4 autoconf/m4/pa_func_attribute_error.m4 autoconf/m4/pa_func_snprintf.m4 autoconf/m4/pa_func_vsnprintf.m4 autoconf/m4/pa_have_func.m4 autoconf/m4/pa_libext.m4 autoconf/m4/pa_sym.m4 autoconf/m4.old/
+ pwd
+ ACLOCAL_PATH=/tmp/nasm/autoconf/m4.old
+ export ACLOCAL_PATH
+ aclocal --install --output=autoconf/aclocal.m4 -I autoconf/m4
./autogen.sh: 38: ./autogen.sh: aclocal: not found
+ test ! -f autoconf/aclocal.m4
+ mv -f autoconf/m4.old/pa_add_cflags.m4 autoconf/m4.old/pa_add_flags.m4 autoconf/m4.old/pa_add_headers.m4 autoconf/m4.old/pa_add_ldflags.m4 autoconf/m4.old/pa_arg_bool.m4 autoconf/m4.old/pa_arg_disabled.m4 autoconf/m4.old/pa_arg_enabled.m4 autoconf/m4.old/pa_c_typeof.m4 autoconf/m4.old/pa_check_bad_stdc_inline.m4 autoconf/m4.old/pa_func_attribute.m4 autoconf/m4.old/pa_func_attribute_error.m4 autoconf/m4.old/pa_func_snprintf.m4 autoconf/m4.old/pa_func_vsnprintf.m4 autoconf/m4.old/pa_have_func.m4 autoconf/m4.old/pa_libext.m4 autoconf/m4.old/pa_sym.m4 autoconf/m4/
+ mv -f autoconf/aclocal.m4.old autoconf/aclocal.m4
mv: cannot stat 'autoconf/aclocal.m4.old': No such file or directory
+ rm -rf autoconf/m4.old
+ autoheader -B autoconf
./autogen.sh: 45: ./autogen.sh: autoheader: not found
+ autoconf -B autoconf
./autogen.sh: 46: ./autogen.sh: autoconf: not found
+ rm -rf autom4te.cache config.log config.status config/config.h Makefile
+ false
./prerequisites.sh: line 170: ./configure: No such file or directory
(nasm) installation has failed.
```

**Comment by getroot** [comment]
We have not tested that prerequisites.sh works well on Debian OS. It is not within our scope of support. It will be supported if people who use it put up a PR.
